plugins/inputs/bigbluebutton/bigbluebutton.go

```diff
@@ -0,0 +1,167 @@
+package bigbluebutton
+
+import (
+	"encoding/xml"
+	"fmt"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type BigBlueButton struct {
+	URL         string `toml:"url"`
+	APIEndpoint string `toml:"api_endpoint"`
+	SecretKey   string `toml:"secret_key"`
+	ServerName  string `toml:"server_name"`
+}
```

Comment:
Please use the internal http package to get the goodies such as TLS, authentication or proxy support. See the [http plugin](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/http/http.go) for an example.

Reply:
Reminder, still open. :-)

Reply:
Just fixed it ! Hope the way I do is fine for you 😊

Reply:
Looks good. Just a question, is it also possible that an authentication is required for gathering the metrics from the server? If so, please also add a username and password option.
Please also remember to add the new config options (for TLS and proxy) to the `sampleConfig` and update the `README.md` file.

Reply:
By default, no. But i'm not 100% sure that someone update nginx configuration and add Basic Authentication. I just added an authentication part.